test-cypress/integration/help/rest-api.spec.js

```diff
@@ -0,0 +1,110 @@
+describe('Help / REST API', () => {
```

Comment:
Isn't this part of the other merge request for swagger/rest? Seems that should not be here.

Reply:
Yes, it is. See https://github.com/Ontotext-AD/graphdb-workbench/pull/49#issuecomment-525644596

Reply:
Rebased and skipped that commit.